src/GGZ-2-0/Cert/XIS-Server/medmij-ggz-xis-1-1-serve.xml

```diff
@@ -300,11 +320,10 @@
         </nts:include>
         <action>
             <assert>
-                <description value="Confirm that the returned searchset Bundle contains 4 entries."/>
+                <description value="Confirm that the returned searchset Bundle contains at least 4 entries."/>
```

Comment:
Misschien een toelichting waarom? Zodat we dat in de toekomst nog weten

Reply:
Done!